Plugins/Flow.Launcher.Plugin.PluginsManager/PluginsManager.cs

```diff
@@ -523,36 +494,6 @@ private void Uninstall(PluginMetadata plugin, bool removedSetting = true)
             using var _ = File.CreateText(Path.Combine(plugin.PluginDirectory, "NeedDelete.txt"));
         }
 
-        private List<Result> AutoCompleteReturnAllResults(string search, string hotkey, string title, string subtitle)
```

Comment:
All of this is changed to `GetDefaultHotkey.Where...` right

Reply:
after some head-scratching, the only use for `AutoCompleteReturnAllResults` that I could find was that it allowed queries like `pm install install install` 😂 

I think it can safely go....